src/UserRepository.js

```diff
@@ -4,7 +4,7 @@ import activityData from './data/activity';
 import userData from './data/users';
 
 class UserRepository {
-  constructor() {
+  constructor(date, userData, activityData, hydrationData, sleepData) {
```

Comment:
To me it makes sense that UserRepository would have knowledge of everyone single person's activity, since its methods are likely going to deal with "best x of all time" etc.